task/kind/0.1/tests/run.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: kind
+spec:
+  pipelineSpec:
+    workspaces:
+      - name: workspace
+    tasks:
+      - name: git
+        taskSpec:
+          steps:
+            - image: docker.io/alpine/git
+              args: ["clone", "https://github.com/wlynch/tekton-kind"]
+              workingDir: $(workspaces.source.path)
+          workspaces:
+            - name: source
```

Comment:
not a blocker but just a thought:- can we not directly use the `git-clone` task for cloning the repo like
https://github.com/tektoncd/catalog/blob/1a38009d97722101792f041edb96c2ea383ec078/task/black/0.1/tests/run.yaml#L10-L22
This will also guarantee that this task is working fine with the git-clone task present in catalog